tests/test_cli_utils.py

```
@@ -767,10 +767,13 @@ async def test_logout_matrix_bot_async_client_none(
         mock_logger = MagicMock()
         mock_get_logger.return_value = mock_logger
 
-        result = await logout_matrix_bot(password="test_password")
+        test_password = "test_password"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "test_password"

(S105)